SupportedServices.md

```diff
@@ -196,6 +197,13 @@ Note that using the defaults here for all alarms is probably not that useful.
 - `LatencyHigh`: 0.5 (average in seconds)
 - `UnHealthyHostCountHigh`: 1 (count)
 
+### Alb
+
+- `Alb5xxErrorsHigh`: 10 (count)
```

Comment:
Can you just remove the Alb prefixes here so they match the  alarms? fine to merge after that